Microsoft Surface Platform-Specific Device Drivers
Say Y here to get to see options for platform-specific device drivers
for Microsoft Surface devices. This option alone does not add any
kernel code.

If you say N, all options in this submenu will be skipped and disabled.
Depends: ARM64 || X86 || COMPILE_TEST
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:20:38.185610300 -0400
@@ -4720,8 +4720,6 @@
 CONFIG_CHARGER_CROS_USBPD=m
 CONFIG_CHARGER_CROS_PCHG=m
 CONFIG_CHARGER_BD99954=m
-CONFIG_BATTERY_SURFACE=m
-CONFIG_CHARGER_SURFACE=m
 CONFIG_BATTERY_UG3105=m
 CONFIG_HWMON=y
 CONFIG_HWMON_VID=m
@@ -7161,15 +7159,6 @@
 #
 CONFIG_AMD_SFH_HID=m
 # end of AMD SFH HID Support
-
-#
-# Surface System Aggregator Module HID support
-#
-CONFIG_SURFACE_HID=m
-CONFIG_SURFACE_KBD=m
-# end of Surface System Aggregator Module HID support
-
-CONFIG_SURFACE_HID_CORE=m
 # end of HID support
 
 CONFIG_USB_OHCI_LITTLE_ENDIAN=y
@@ -8199,20 +8188,7 @@
 CONFIG_MLXREG_IO=m
 CONFIG_MLXREG_LC=m
 # CONFIG_NVSW_SN2201 is not set
-CONFIG_SURFACE_PLATFORMS=y
-CONFIG_SURFACE3_WMI=m
-CONFIG_SURFACE_3_POWER_OPREGION=m
-CONFIG_SURFACE_ACPI_NOTIFY=m
-CONFIG_SURFACE_AGGREGATOR_CDEV=m
-CONFIG_SURFACE_AGGREGATOR_REGISTRY=m
-CONFIG_SURFACE_DTX=m
-CONFIG_SURFACE_GPE=m
-CONFIG_SURFACE_HOTPLUG=m
-CONFIG_SURFACE_PLATFORM_PROFILE=m
-CONFIG_SURFACE_PRO3_BUTTON=m
-CONFIG_SURFACE_AGGREGATOR=m
-CONFIG_SURFACE_AGGREGATOR_BUS=y
-# CONFIG_SURFACE_AGGREGATOR_ERROR_INJECTION is not set
+# CONFIG_SURFACE_PLATFORMS is not set
 CONFIG_HAVE_CLK=y
 CONFIG_HAVE_CLK_PREPARE=y
 CONFIG_COMMON_CLK=y
